cypress/e2e/moviesGrid.cy.js

```diff
@@ -2,21 +2,103 @@ import allMoviesData from '../fixtures/all-movies-data.json'
 
 describe('Main Page', () => {
   beforeEach(() => {
-    cy.visit('http://localhost:3000');
+    cy.visit('http://localhost:3000/')
   });
 
-  it('should display all movies when the page has loaded', () => {
-    cy.intercept('https://rancid-tomatillos.herokuapp.com/api/v2/movies', allMoviesData)
-    cy.get('.movieCard').should('have.length', 40)
-  })
+  describe("Main Page Loading", () => {
+    it("Should display a loading page while fetching movie data", () => {
+      cy.intercept('https://rancid-tomatillos.herokuapp.com/api/v2/movies', allMoviesData).visit('http://localhost:3000/')
```

Comment:
I would make this whole line an intercept command so you don't repeat it in every test. use this in the beforeEach and remove from the tests. this will DRY up our tests.